server/hub.go

```diff
@@ -130,6 +134,8 @@ func newHub() *Hub {
 	statsRegisterInt("FileDownloadsTotal")
 	statsRegisterInt("FileUploadsTotal")
 
+	statsRegisterHistogram("RequestLatency", RequestLatencyDistribution[:]...)
```

Comment:
The `statsRegisterHistogram` can just take `RequestLatencyDistribution` as a second parameter. I see no value in making `statsRegisterHistogram` variadic.